bot.py

```diff
@@ -28,14 +28,15 @@
 
 TODO: manual game creation
 """
-db_handler = DatabaseHandler(host=os.environ.get('DB_HOST'), db_name=os.environ.get('DB_NAME'), user=os.environ.get('DB_USER'), password=os.environ.get('DB_PASS'))
+db_handler = DatabaseHandler(host=os.environ.get('DB_HOST'), db_name=os.environ.get(
+    'DB_NAME'), user=os.environ.get('DB_USER'), password=os.environ.get('DB_PASS'))
```

Comment:
nit: What's up with the spacing on these lines? Might just be a github render thing

Reply:
My VS code decided to auto format itself